client/jetpack-cloud/sections/partner-portal/primary/prices/index.tsx

```diff
@@ -0,0 +1,138 @@
+		const dailyUserYearlyPrice = userYearlyProduct ? userYearlyProduct.cost / 365 : 0;
+
+		return (
+			<tr key={ product.product_id }>
+				<td>
+					<strong>{ product.name }</strong>
+					<LicenseBundleCardDescription product={ product } />
+				</td>
+				<td>
+					<div className="prices__mobile-description">
+						{ translate( 'Jetpack.com Pricing, purchased Monthly (USD)' ) }
+					</div>
+					<div>
+						{ translate( '%(price)s/day', {
+							args: {
+								price: formatCurrency( dailyUserMonthlyPrice, 'USD', currencyFormatOptions ),
+							},
+						} ) }
+					</div>
+					<div>
+						{ translate( '%(price)s/month', {
+							args: {
+								price:
+									userMonthlyProduct &&
+									formatCurrency( userMonthlyProduct.cost, 'USD', currencyFormatOptions ),
+							},
+						} ) }
+					</div>
+				</td>
+				<td>
+					<div className="prices__mobile-description">
+						{ translate( 'Jetpack.com Pricing, purchased Yearly (USD)' ) }
+					</div>
+					<div>
+						{ translate( '%(price)s/day', {
+							args: {
+								price: formatCurrency( dailyUserYearlyPrice, 'USD', currencyFormatOptions ),
+							},
+						} ) }
+					</div>
+					<div>
+						{ translate( '%(price)s/year', {
+							args: {
+								price:
+									userYearlyProduct &&
+									formatCurrency( userYearlyProduct.cost, 'USD', currencyFormatOptions ),
+							},
+						} ) }
+					</div>
+				</td>
+				<td>
+					<div className="prices__mobile-description">
+						{ translate( 'Agency/Pro Pricing, purchased daily (USD)' ) }
+					</div>
+					{ translate( '%(price)s/day', {
+						args: {
+							price:
+								dailyAgencyPrice &&
+								formatCurrency( dailyAgencyPrice, 'USD', currencyFormatOptions ),
+						},
+					} ) }
+				</td>
+			</tr>
+		);
+	} );
+
+	return (
+		<Main wideLayout className="prices">
+			<QueryProductsList type="jetpack" />
+
+			<DocumentHead title={ translate( 'Prices' ) } />
+			<SidebarNavigation />
+
+			<div className="prices__header">
+				<CardHeading size={ 36 }>
+					{ translate( 'Jetpack Agency & Pro Partner Program Product Pricing' ) }
+				</CardHeading>
+
+				<SelectPartnerKeyDropdown />
+			</div>
+
+			<div className="prices__description">
+				<p>
+					{ translate(
+						'The following products are available through the Licenses section. Prices are calculated daily and invoiced at the beginning of the next month. Please note that the Jetpack pro Dashboard prices will be displayed as a monthly cost. If you want to determine a yearly cost for the Agency/Pro pricing, you can take the daily cost x 365.'
+					) }
+				</p>
+			</div>
+
+			<table className="prices__table">
+				<tbody>
+					<tr>
+						<td></td>
+						<td>{ translate( 'Jetpack.com Pricing, purchased Monthly (USD)' ) }</td>
+						<td>{ translate( 'Jetpack.com Pricing, purchased Yearly (USD)' ) }</td>
+						<td>{ translate( 'Agency/Pro Pricing, purchased daily (USD)' ) }</td>
```

Comment:
Nitpick: table headers are best represented with `<th>` instead of `<td>`. For most users, it won't matter, but assistive and text-to-speech software might interpret them differently. However, if you decide to replace the `td`s with `th`s, you will have to update your SCSS rules to ensure the style choices are applied to them as well.